### DISCUSSION TITLE
Chromecast Receiver Muted

### DISCUSSION THREAD

**Discussion by jeremykuang** [PierfrancescoSoffritti/android-youtube-player#1134] [Q&A]
Hi everyone! I tried to the Chromecast Receiver but I was not able to play the YouTube video until it is muted. But when I unmute it, it is paused and I can't play the video again until I mute it. Do you experience the same problem? If you do, how do you all overcome it?

**Comment by PierfrancescoSoffritti** [comment]
This is probably the reason why: https://developer.chrome.com/blog/autoplay
